perllib/FixMyStreet/App/Form/Claims.pm

```diff
@@ -339,6 +339,7 @@ has_field incident_time => (
     required => 1,
     type => 'Text',
     html5_type_attr => 'time',
+    tags => { hint => 'For example 06:30 PM or 18:30 depending on your settings' },
```

Comment:
Probably need to make the control wider too as, in chrome at least, you can't see the am/pm bit:

<img width="559" alt="Screenshot 2021-08-03 at 11 40 11" src="https://user-images.githubusercontent.com/5310/128002569-27be9786-065a-48d3-8385-60d1a1d3a8aa.png">

Reply:
I've had a look at this and seems like our time field size is set by the `base.css`. Though I could change this, is does seem like that could have a wider system effect and perhaps I should raise the issue with the design team?

Reply:
looks like there's something for it in _claims.scss